external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -29,7 +29,7 @@ import org.apache.spark.tags.DockerTest
 @DockerTest
 class PostgresIntegrationSuite extends DockerJDBCIntegrationSuite {
   override val db = new DatabaseOnDocker {
-    override val imageName = "postgres:11.4"
+    override val imageName = "postgres:12.0-alpine"
```

Comment:
`12.0-alpine` instead of `12.0`?

Reply:
Yes. This is another improvement. It reduces the image size a lot. This is good for CI/CI testing environment.
```
$ docker images | grep postgres
postgres                         12.0-alpine          5b681acb1cfc        2 days ago          72.8MB
postgres                         11.4                 53912975086f        3 months ago        312MB
```

Reply:
I'll add this to the PR description.

Reply:
Thanks! Yea, it looks nice.